resources/openapi_v2.yaml

```diff
@@ -7,20 +7,23 @@ info:
 paths:
   /applications:
     get:
-      description: Return all the applications related to a user or the apointted ministry
+      description: Return all the applications related to a user or the appointed ministry using the authentication token and admin or ministry can query the application by user-id or ministry-id and user can query the application using draft.
```

Comment:
This description is too hard to understand in my point of view. Let's reduce the word count and make this statement understandable. @sanjulamadurapperuma your thoughts on this?

Reply:
Okay. Any suggestions? 
"Return all the applications related to a user or appointed ministry. Admin can query the applications by user-id and ministry-id. Ministry can query the applications by user-id"

Reply:
"Return all the applications related to a user or the appointed ministry. The ministry can query the applications using the user-id and the administrators are able to query applications from either the user-id or the ministry-id."